src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -155,12 +155,24 @@
                  (str/join "', '" (sort valid-methods)) "'"))))
   (when (= "regex" (-> rule :match-request :type name str/lower-case))
     (try
-      (re-pattern (:path (:match-request rule)))
+      (let [pattern (re-pattern (:path (:match-request rule)))
+            group-count (-> pattern (.matcher "") .groupCount)
```

Comment:
Admittedly, it's a hack I found on Stack Overflow, but it seems to get the job done.